docs/source/release_notes.rst

```diff
@@ -2,6 +2,26 @@ Release History
 ###############
 
 
+v4.1.0 (2024-12-19)
```

Comment:
This could also be v4.0.1 because it's only bugfixes but it's not a meaningful distinction